---
description: 'Saiba mais sobre: System. ServiceModel. AsyncCallbackThrewException'
title: System.ServiceModel.AsyncCallbackThrewException
ms.date: 03/30/2017
ms.assetid: dfc27ca7-b904-4f29-a5b6-dc702fa54a20
ms.openlocfilehash: 53ef853d22041099eb68fe79f7ef43bc24c55798
ms.sourcegitcommit: ddf7edb67715a5b9a45e3dd44536dabc153c1de0
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 02/06/2021
ms.locfileid: "99644691"
---
# <a name="systemservicemodelasynccallbackthrewexception"></a>System.ServiceModel.AsyncCallbackThrewException

System.ServiceModel.AsyncCallbackThrewException  
  
## <a name="description"></a>Descrição  

 Um retorno de chamada assíncrono gerou uma exceção.  
  
## <a name="see-also"></a>Consulte também

- [Rastreamento](index.md)
- [Utilizando o rastreamento para solucionar problemas em seu aplicativo](using-tracing-to-troubleshoot-your-application.md)
- [Administração e diagnósticos](../index.md)
